### PR TITLE
completed javascript-timers - brighitte johnson

### PR DESCRIPTION
**Pull request by brij8**
javascript-timers01
javascript-timers02

### CODE REVIEW

File: javascript-timers/interval.js [hunk 1]
```diff
@@ -0,0 +1,16 @@
+setInterval(bowie, 1000);
```
Comment (Zhiwutian) [inline javascript-timers/interval.js:1]: make sure you are saving the `intervalId` into a variable and passing it to `clearInterval` when you are done with the interval

File: javascript-timers/interval.js [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+  if (h1msg.textContent === '4') {
+    h1msg.textContent = '3';
+  } else if (h1msg.textContent === '3') {
+    h1msg.textContent = '2';
+  } else if
+  (h1msg.textContent === '2') {
+    h1msg.textContent = '1';
+  } else if
+  (h1msg.textContent === '1') {
+    h1msg.textContent = '~Earth Beeeelooowww Us~';
```
Comment (Zhiwutian) [inline javascript-timers/interval.js:14]: this would be where to call clearInterval

**Review by Zhiwutian — CHANGES_REQUESTED** [review]
update needed

**Review by Zhiwutian — APPROVED** [review]
👍🏼

**Comment by brij8** [comment]
like so?

**Pull request merged**